service/geopmdpy/service.py

```diff
@@ -817,6 +833,21 @@ def pop_profile_region_names(self, profile_name):
             result.sort()
         return result
 
+    def log_access(self):
+        """Print all signals and controls that have been accessed
+
+        """
+        if len(self._accessed_signals) != 0:
```

Comment:
Might be good to put this whole block in a try/catch since it's called from the finally clause in main.py.

Reply:
It did it at the call site